mailing_list/api.py

```diff
@@ -41,6 +57,11 @@ def lists(request, resource_link_id):
                 'modified_by': logged_in_user_id
             }
         )
+
+        if has_lti_role(request, const.LEARNER):
+            # Learners should only see lists which they can post to, so filter the mailing lists
+            custom_canvas_user_id = int(request.session['LTI_LAUNCH']['custom_canvas_user_id'])
```

Comment:
nitpick: the "custom_" prefix is to namespace is within the lti launch parameters, right?  shouldn't this just be canvas_user_id?